lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+[
+    {
+        "ISO3166-1-Alpha-3": "TWN",
+        "ISO3166-1-Alpha-2": "TW",
+        "official_name_fr": "",
+        "ISO4217-currency_name": "",
+        "ISO4217-currency_alphabetic_code": "",
+        "official_name_ar": "",
+        "official_name_es": "",
+        "official_name_cn": "",
+        "official_name_en": "",
```

Comment:
`Taiwan`

Reply:
this was missing in source data for some reason, i didn't validate it

https://github.com/datasets/country-codes/blob/master/data/country-codes.csv